ndmg/utils/qa_utils.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+# Copyright 2016 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# qa_utils.py
+# Created by Wilson Tang
+
+import numpy as np
+
+def get_min_max(data, minthr=2, maxthr=95):
+    """
+    Parameters
+    -----------
+    data: np array
+        3-d regmri data to threshold.
+    minthr: int
+    maxthr: int
```

Comment:
give `minthr` and `maxthr` actual descriptions

Reply:
give the function an actual description as well

Reply:
check commit 09e7135